ro_help/hub/urls.py

```diff
@@ -1,5 +1,4 @@
 from django.urls import path, include
-from django.views.generic import TemplateView
```

Comment:
This is used by static pages

Reply:
This doesn't seem to be used there though. Unless i'm missing something.

Reply:
You are right. It's the other urls.py file that uses it :)